models/schema_template_vrf.go

```diff
@@ -11,15 +11,16 @@ func NewSchemaTemplateVrf(ops, path, Name, displayName, ipDataPlaneLearning stri
 
 	if ops != "remove" {
 		VrfMap = map[string]interface{}{
-			"displayName":         displayName,
-			"name":                Name,
-			"l3MCast":             l3m,
-			"vzAnyEnabled":        vzany,
-			"ipDataPlaneLearning": ipDataPlaneLearning,
-			"preferredGroup":      preferredGroup,
+			"displayName":    displayName,
+			"name":           Name,
+			"l3MCast":        l3m,
+			"vzAnyEnabled":   vzany,
+			"preferredGroup": preferredGroup,
+		}
+		if ipDataPlaneLearning != "" {
```

Comment:
Should we do the same conditional for al non-required parameters?

Reply:
I tested for all the other non-required parameters, and they work without any error, even if it is empty, since most of them are boolean.

Reply:
ok!